lib/granite/form/config.rb

```diff
@@ -27,16 +27,16 @@ def normalizer(name, &block)
         end
       end
 
-      def typecaster(*classes, &block)
-        classes = classes.flatten
-        if block
-          _typecasters[classes.first.to_s.camelize] = block
-        else
-          _typecasters[classes.detect do |klass|
-            _typecasters[klass.to_s.camelize]
-          end.to_s.camelize] or raise TypecasterMissing, classes
+      def typecaster(class_name, &block)
```

Comment:
`typecaster` is now a quick shortcut to define a type that only needs typecasting.
To access existing types you need to use `type_for` or `types`